pilz_control/include/pilz_control/pilz_joint_trajectory_controller_impl.h

```diff
@@ -74,13 +74,16 @@ bool PilzJointTrajectoryController<SegmentImpl, HardwareInterface>::is_executing
 
   bool is_executing {false};
 
+  typedef joint_trajectory_controller::SegmentTolerancesPerJoint<typename Segment::Scalar> SegmentTolerancesPerJoint;
```

Comment:
I suggest to use the "new" alias-decleration. See [nextptr.com](https://www.nextptr.com/tutorial/ta1193988140/how-cplusplus-using-or-aliasdeclaration-is-better-than-typedef)

Is it maybe even possible to skip the `typedef`if you use `auto`?

Reply:
Using `auto` works.